Source/relay/TourGuide/Items of the Month/2023/SIT Course Certificate.ash

```diff
@@ -0,0 +1,35 @@
+boolean hasAnySkillOf(string [int] skillNames) {
```

Comment:
cute lil foreach. i'm merging this because i'm about to merge in your trainset change into the release but there's probably a mild case to make this a utils.ash thing instead of just a thing within the SIT ash file. i do not care enough to wait for this change to merge it up tho.

Reply:
I thought about this too. In TS I would have just done a filter/reduce/etc. Maybe we should add more utils for any/all/etc for a bunch of different types 🤷